contracts/src/BeefyClient.sol

```diff
@@ -91,12 +94,14 @@ contract BeefyClient {
      * @param bitfield a bitfield signalling which validators they claim have signed
      * @param blockNumber the block number for this commitment
      * @param validatorSetLen the length of the validator set for this commitment
+     * @param signatureCountRequired the number of signatures required
      * @param bitfield a bitfield signalling which validators they claim have signed
      */
     struct Ticket {
         address account;
         uint64 blockNumber;
         uint32 validatorSetLen;
+        uint256 signatureCountRequired;
```

Comment:
This field name sounds like its a boolean rather than a number.

```suggestion
        uint256 numRequiredSignatures;
```

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/956/commits/46250d8ff06a500a194a2c57acd59397e343af46